### ISSUE TITLE
ncdu: Ability to see and sort on date/time

### ISSUE THREAD

**Issue by rawtaz** [rclone/rclone#6537]
#### What is your current rclone version (output from `rclone version`)?

~~~
rclone v1.60.0
- os/version: darwin 10.14.6 (64 bit)
- os/kernel: 18.7.0 (x86_64)
- os/type: darwin
- os/arch: amd64
- go/version: go1.19.2
- go/linking: dynamic
- go/tags: cmount
~~~

#### What problem are you are trying to solve?

Browse files in a useful manner :)

I have an Azure Blob Storage container in which there are a lot of files, all named with a SHA256 hash. The Azure portal shows columns for the name of course, and also the modification time.

I would like to sort the list on modification time, such that I can see the most recent files without having to scroll a ton of pages down while at the same time waiting for it to load more lines over and over again.

Unfortunately, one cannot sort the list in the Azure portal, it just lists all of the blobs/objects by their name, which when they're all hashes is totally useless to see the most recent ones.

#### How do you think rclone should be changed to solve that?

I tried using `rclone ncdu` to view the list in a better way, but it seems that it 1) does not show any date/time info for the blobs/objects, and 2) does not allow to sort the list on a date/time.

The feature request is simple: Make ncdu able to show date/time information (at least modification time, but also others if there are more), and also enable the user to sort the list on these dates/times.

#### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

**Comment by albertony** [comment]
I had a quick go at this, see pull request #6539.
I could trigger a beta build, if you are willing to test the changes on a beta version?

**Comment by ncw** [comment]
I've merged this to master now which means it will be in [the latest beta](https://beta.rclone.org/) in 15-30 minutes and released in v1.61
